i18n/management/commands/log_i18n_failure.py

```diff
@@ -9,4 +9,4 @@ class Command(BaseCommand):
     See bin/sync_i18n.sh for context.
     """
     def handle(self, *args, **options):
-        log("ERROR: I18n Sync encountered a problem and did not complete")
+        log("<!subteam^S017CFFHHT9|foundations-team> ERROR: I18n Sync encountered a problem and did not complete")
```

Comment:
Is this `subteam^S017CFFHHT9` the slack-id for FND team? How do you find it?

Reply:
https://api.slack.com/methods/usergroups.list/test
You can find the `id` for the `foundations-team` in the response from the `usergroups.list` API